### PR TITLE
feat(response): add new middleware

### PR DESCRIPTION
**Pull request by ya-woodcutter**

### CODE REVIEW

File: src/response.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    key: 'response'
+};
+
+export default function responseMw(settings) {
+    settings = Object.assign({}, defaultSettings, settings);
+
+    return next => async options => {
+        const result = await next();
+
+        if (isAPIError(result) || options[settings.key]) {
```
Comment (evilj0e) [inline src/response.js]: For what reason? `options[settings.key]`

Comment (ya-woodcutter) [reply src/response.js]: Because in all middlewares we support this option :)

File: src/response.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ export default function responseMw(settings) {
     return next => async options => {
         const result = await next();
 
-        if (isAPIError(result) || options[settings.key]) {
+        if (isAPIError(result) || (options && options[settings.key])) {
```
Comment (evilj0e) [inline src/response.js]: It seems you made mistake here. Did you mean `!options || !(options && options[settings.key])`?

Comment (evilj0e) [reply src/response.js]: It's my fault. You are right here.

**Review by evilj0e — APPROVED** [review]

**Pull request merged**